modules/installation-mirror-repository.adoc

```diff
@@ -244,10 +251,63 @@ content.
 
 You must perform this step on a machine with an active internet connection.
 ====
+
```

Comment:
*PEER REVIEWER* The following added lines are not new text. The steps are copied from above and some substeps removed.